xdp-dump/tests/test-xdpdump.sh

```diff
@@ -448,11 +448,11 @@ test_promiscuous_selfload()
     fi
 
     RESULT=$(dmesg)
-    if [[ "$RESULT" != *"device $NS entered promiscuous mode"* ]]; then
+    if [[ "$RESULT" != *"device $NS entered promiscuous mode"* && "$RESULT" != *"$NS: entered promiscuous mode"* ]]; then
```

Comment:
Better to combine these as two distinct 'test' invocations, like:
`    if [[ "$RESULT" != *"device $NS entered promiscuous mode"* ]] && [[ "$RESULT" != *"$NS: entered promiscuous mode"* ]]; then`

same for all of the checks below, obviously :)

Reply:
Thanks, @tohojo I updated and tested the patch as your suggestion.